core-bundle/src/Resources/public/sectionwizard.js

```diff
@@ -0,0 +1,101 @@
+(function(){
+    'use strict';
+
+    const initialized = [];
+
+    const init = (node) => {
+        node.querySelectorAll('.tl_sectionwizard').forEach((table) => {
+            if (initialized.includes(table)) {
+                return;
+            }
+
+            initialized.push(table);
```

Comment:
```suggestion
            if (initialized.has(table)) {
                return;
            }

            initialized.set(table, true);
```

Reply:
👍 however, do we actually need to keep track of initialized nodes? I only included it because https://github.com/terminal42/contao-conditionalformfields/blob/develop/public/conditionalformfields.js does it this way as well 🙈

Reply:
Yes, I think we have to because with the mutation observer it could be that changes in the tree trigger it again.
Unless there is another way to detect if the script did already run for a certain table?

Reply:
We could also set an arbitrary boolean property on the node itself - that's what I usually do these days, but not sure what is best practise.

Reply:
I think setting arbitrary properties on nodes is considered bad because of possible naming conflicts.